### PR TITLE
feat: add Swahili translations

### PR DESCRIPTION
**Pull request by Dukent2903**
added Swahili translation.
image

### CODE REVIEW

File: src/locales/en/common.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "languages": {
     "en": "English",
     "fr": "French",
-    "ar": "Arabic"
+    "ar": "Arabic",
+    "sw": "Swahili"
```
Comment (HugoPerard) [inline src/locales/en/common.json:6]: This add is missing in fr/common.json and in ar/common.json

**Comment by ivan-dalmet** [comment]
Hi @Dukent2903 thanks for the contribution.

To prevent any issue in the future, can you add a section in the CONTRIBUTING.md file about translations with the following content:

```
## Translations

You only need to maintain the english and french translations for you PR. 
Other language translations are not required to submit a PR 😉
```

File: src/locales/en/common.json (hunk 1 above)
Comment (Dukent29) [reply src/locales/en/common.json:6]: i corrected it, it is now available

**Comment by ivan-dalmet** [comment]
Thanks for the update @Dukent2903, can you run the `yarn pretty` command to fix the styling issue?

**Review by yoannfleurydev — APPROVED** [review]
Looks good to me

bitmoji

**Pull request merged**